### PR TITLE
Doro67 Multi PCB port

### PR DESCRIPTION
**Pull request by ShadeDream**
## Description

New keyboard configuration and default keymaps for the Doro67 multi PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/doro67/multi/keymaps/default_multi/keymap.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include QMK_KEYBOARD_H
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_multi(
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_F2,   KC_INS,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,
+    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  KC_PGUP,
+    KC_LSFT, KC_F1,   KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP, KC_PGDN,
```
Comment (vomindoraan) [inline keyboards/doro67/multi/keymaps/default_multi/keymap.c]: `KC_F1` is a bit of an odd choice. Is that what came pre-flashed on the PCB?

File: keyboards/doro67/multi/keymaps/default/readme.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+```
+,---------------------------------------------------------------.
+|Esc|  1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|BackSp |Ins|
+|---------------------------------------------------------------|
+|Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |Del|
+|---------------------------------------------------------------|
+|Ctrl  |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '| Enter  |PUp|
+|---------------------------------------------------------------|
+|Shift    |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift| Up|PDn|
```
Comment (noroadsleft) [inline keyboards/doro67/multi/keymaps/default/readme.md]: Key alignment.

```suggestion
|Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /| Shift| Up|PDn|
```

File: keyboards/doro67/multi/keymaps/default_multi/readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```
+,---------------------------------------------------------------.
+|Esc|  1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =| BS| F2|Ins|
+|---------------------------------------------------------------|
+|Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |Del|
+|---------------------------------------------------------------|
+|Ctrl  |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '| Enter  |PUp|
+|---------------------------------------------------------------|
+|Shft|  F1|  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift| Up|PDn|
```
Comment (noroadsleft) [inline keyboards/doro67/multi/keymaps/default_multi/readme.md]: Key alignment.

```suggestion
|Shft| F1|  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /| Shift| Up|PDn|
```

File: keyboards/doro67/multi/keymaps/default_multi/readme.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+```
+,---------------------------------------------------------------.
+|`  | F1| F2| F3| F4| F5| F6| F7| F8| F9|F10|F11|F12|       |   |
+|---------------------------------------------------------------|
+|BLTog|Stp|Dec|Inc|   |   |   |   |   |   |   |   |   |     |   |
+|---------------------------------------------------------------|
+|      |   |   |   |   |   |   |   |   |   |   |   |        |   |
+|---------------------------------------------------------------|
+|        |   |   |   |   |   |   |   |   |   |   |      |   |   |
```
Comment (noroadsleft) [inline keyboards/doro67/multi/keymaps/default_multi/readme.md]: Key alignment.

```suggestion
|    |   |   |   |   |   |   |   |   |   |   |   |      |   |   |
```

File: keyboards/doro67/multi/keymaps/default_multi/readme.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+|---------------------------------------------------------------|
+|     |    |     |                       |     |    |   |   |   |
```
Comment (noroadsleft) [inline keyboards/doro67/multi/keymaps/default_multi/readme.md]: Key alignment.

```suggestion
|     |    |     |        |     |       |   |   |   |   |   |   |
```

File: keyboards/doro67/multi/readme.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Doro67 Multi PCB
+===
+
+A custom 65% keyboard with multiple layout support.
+
+Do not flash RGB firmware for this board. 
+
+Keyboard Maintainer: [ShadeDream](https://github.com/shadedream)  
+Hardware Supported: Doro67 Multi PCB
```
Comment (noroadsleft) [inline keyboards/doro67/multi/readme.md]: This line needs two spaces at the end for GitHub to render it properly.

```suggestion
Hardware Supported: Doro67 Multi PCB  
```

File: keyboards/doro67/multi/keymaps/default_multi/keymap.c (hunk 1 above)
Comment (ShadeDream) [reply keyboards/doro67/multi/keymaps/default_multi/keymap.c]: Yep. Those are the defaults when it comes to you (and in their json file). We could set them to something more appropriate, but I'm not sure what people map those to typically and if it'll throw off the rest of the keymap more. I did make some very small additions and modifications to the ANSI and ISO keymaps to make them more usable out of the "box".

Comment (mechmerlin) [reply keyboards/doro67/multi/keymaps/default_multi/keymap.c]: I'd split the backspace to hhkb style on top, for the F1 maybe make it what the iso default is which is `KC_NUBS`

**Comment by noroadsleft** [comment]
Now that we're getting another Doro67 PCB added, `keyboards/doro67/readme.md` should probably be fleshed out a bit; it's zero-length right now.

**Comment by mechmerlin** [comment]
> Now that we're getting another Doro67 PCB added, `keyboards/doro67/readme.md` should probably be fleshed out a bit; it's zero-length right now.

We'll do that once we have all three or so PCBs accounted for.

File: keyboards/doro67/multi/multi.h [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ansi( \
+  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
+  K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, \
+  K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D, K2E, \
+  K30, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D, K3E, \
+  K40, K41, K42, K44, K49, K4A, K4C, K4D, K4E \
```
Comment (vomindoraan) [inline keyboards/doro67/multi/multi.h]: How about aligning the arguments so they match the physical layout more closely? Kind of like what you did in `default/keymap.c` and `default_iso/keymap.c`?

Comment (mechmerlin) [reply keyboards/doro67/multi/multi.h]: I actually don't like it when its aligned to match the physical layout.

**Comment by drashna** [comment]
Looks good, thanks!

**Pull request merged**